app/src/app/[lng]/cities/[cityId]/HIAP/HiapPageLayout.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box } from "@chakra-ui/react";
+import { Hero } from "@/components/GHGIHomePage/Hero";
+import { InventoryResponse, CityWithProjectDataResponse } from "@/util/types";
+import { PopulationAttributes } from "@/models/Population";
+import { TFunction } from "i18next";
+
+interface HiapPageLayoutProps {
+  inventory: InventoryResponse | null;
+  formattedEmissions: { value: string; unit: string };
+  lng: string;
+  population: PopulationAttributes | null;
+  children: React.ReactNode;
+  city?: CityWithProjectDataResponse | undefined;
+}
+
+export function HiapPageLayout({
+  inventory,
+  formattedEmissions,
+  lng,
+  population,
+  children,
+  city,
+}: HiapPageLayoutProps) {
+  return (
+    <Box
+      h="full"
+      display="flex"
+      flexDirection="column"
+      bg="background.backgroundLight"
+    >
```

Comment:
You can use the <VStack> component to replace the display and flexDirection properties here.